### DISCUSSION TITLE
VIDEO SCRIPT, Saving file as /artist/artist - tittle.m4a

### DISCUSSION THREAD

**Discussion by bbrodka2** [RandomNinjaAtk/arr-scripts#222] [Q&A] [answered]
is there a way to save video files a .m4a instead of default mkv? for better plex medadata intergration? It is my understanding plex does not use metadata from mkv using video - other library

I would also like t set filename to artist - tittle.m4a

Thanks to any who can help

**Comment by RandomNinjaAtk** [comment, marked as answer]
No, it currently doesn't support saving as m4a. The reason for not supporting it is it doesn't support every codec that is available and depending on the source the codecs can vary... 

As far as the file naming, it's already designed to work with Plex, which it does pretty well based on my own experience. Plex doesn't really support metadata for music videos other than artist name and track title.....
